### ISSUE TITLE
The "unique" column option is not supported

### ISSUE THREAD

**Issue by arsenty** [thrillfall/nextcloud-gpodder#2]
When I try to enable app, got error:
> An error occured during the request. Unable to proceed.
> The "unique" column option is not supported.

Probably this is because I use PostgreSQL?

**Comment by thrillfall** [comment]
My development setup uses sqlite file database and i got "index name is too long" error. I hope i fixed that in the latest release.
I did not test with any other database yet

**Comment by thrillfall** [comment]
So it does work with sqlite and mariadb. So i'd support your assumption this being related to postgres

**Comment by spiezmaestro** [comment]
Can confirm. Get the same message, am also on postgresql.

**Comment by thrillfall** [comment]
Lmgtfy gives us this: https://github.com/nextcloud/server/issues/25607#issuecomment-826596639

The proposed solution is already implemented. But the (old and now broken) way of defining unique column is also still implemented. 
Removing the latter should do the trick. Can s.b. please try and let me know?

TL;DR; does removing lib/Migration/Version0001Date20210520063113.php:L30 help?

**Comment by arsenty** [comment]
@thrillfall, did it and got new error:
<img width="614" alt="image" src="https://user-images.githubusercontent.com/3749464/124873735-87881980-dfcf-11eb-89f1-129708baa03f.png">
but I updated to NC22 yesterday.

**Comment by thrillfall** [comment]
> @thrillfall, did it and got new error:
> <img alt="image" width="614" src="https://user-images.githubusercontent.com/3749464/124873735-87881980-dfcf-11eb-89f1-129708baa03f.png">
> but I updated to NC22 yesterday.

thanks. i just released 1.0.2 with the fix for the unique issue

@arsenty can you please create a new issue for the new error. thanks
